src/open_inwoner/plans/tests/test_htmx.py

```diff
@@ -0,0 +1,80 @@
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from playwright.sync_api import expect
+
+
+def live_reverse(name, kwargs=None, star=False):
+    """
+    Optionally append wildcard to Playwright's `wait_for_url` function to work with plan.uuid's.
+    """
+    url = reverse(name, kwargs=kwargs or {})
+    return f"{url}{'*' if star else ''}"
+
+
+class TestPlansHTMX:
+    uploaded_bytes = b"fake pdf"
+    uploaded_filename = "upload_test.pdf"
+
+    def test_plan_file_upload_e2e(
+        self, live_server, browser, plan_factory, user, user_login_state
+    ):
```

Comment:
`browser` comes from from `PlaywrightSyncLiveServerTestCase` if you inherit from it. However, these are not values for test method params, but attributes on the test instance. Hence remove these params and inside the test: `self.browser.do_stuff()`. Similarly, the `user` should be created in the setup for the test class and accessed with `self.user`. See https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/accounts/tests/test_action_views.py for an example